### PR TITLE
Small design fixes

### PR DESCRIPTION
**Pull request by ahmedqarmout2**

### CODE REVIEW

**Comment by esteeroramy** [comment]
Screenshots/gifs please

**Review by mehtadarshan1 — APPROVED** [review]

File: public/css/leaderboard.css [hunk 1]
```diff
@@ -19,7 +19,6 @@ img#first {
     display: inline-flex;
     max-width: 60%;
     width: 250px !important;
-    height: 250px !important;
```
Comment (esteeroramy) [inline public/css/leaderboard.css:22]: why are we removing this?

File: public/js/analytics.js [hunk 2]
```diff
@@ -23,7 +23,16 @@ $(function () {
   $('#nav-analytics').addClass('active');
 
   // Loads the Student statistics by default
-  displayStudentStatistics(null);
+  if ($('#class-analytics-card').length) {
```
Comment (esteeroramy) [inline public/js/analytics.js]: what property does this check for? this seems like more of a hack that we should not be doing

File: public/js/analytics.js [hunk 3]
```diff
@@ -23,7 +23,16 @@ $(function () {
   $('#nav-analytics').addClass('active');
 
   // Loads the Student statistics by default
```
Comment (esteeroramy) [inline public/js/analytics.js]: this comment wouldnt make sense based on your changes

File: server.js [hunk 4]
```diff
@@ -572,7 +583,7 @@ app.get('/statistics', function (req, res) {
             }
 
             questionslist.forEach(question => {
-      	        var first = studentslist.find(student => {
+                  var first = studentslist.find(student => {
```
Comment (esteeroramy) [inline server.js]: spacing is off here

File: views/student-analytics.pug [hunk 5]
```diff
@@ -1,267 +1,281 @@
-#student-analytics-card
-  .card.white.padding-left-10.padding-right-10
+.container.hidden#student-analytics-card
+  if isAdmin
```
Comment (esteeroramy) [inline views/student-analytics.pug:2]: why are we moving this here, were there any problems where it was previously?

**Review by esteeroramy — CHANGES_REQUESTED** [review]
see comments

File: public/css/leaderboard.css (hunk 1 above)
Comment (ahmedqarmout2) [reply public/css/leaderboard.css:22]: this was causing the resize issue

File: public/js/analytics.js (hunk 2 above)
Comment (ahmedqarmout2) [reply public/js/analytics.js]: it checks if the student has access to class analytics, aka if the student is admin

File: public/css/leaderboard.css (hunk 1 above)
Comment (esteeroramy) [reply public/css/leaderboard.css:22]: picture pls

File: views/student-analytics.pug (hunk 5 above)
Comment (ahmedqarmout2) [reply views/student-analytics.pug:2]: it was easier to make the default class that way

File: public/js/analytics.js (hunk 2 above)
Comment (esteeroramy) [reply public/js/analytics.js]: seems like a hack, please find a better way

Comment (ahmedqarmout2) [reply public/js/analytics.js]: yes I can send an extra ajax request just to get the permission and to make you happy, would you like me to do so?

Comment (ahmedqarmout2) [reply public/js/analytics.js]: there are a lot other ways to check that, navbar options for questions and feedback, fixing the pug to handle different boards, change the js files to split the boards, I just picked a simple if statement

Comment (ahmedqarmout2) [reply public/js/analytics.js]: if we had time to implement the me object we wouldnt have ran into this issue, and a lot other issue

Comment (ahmedqarmout2) [reply public/js/analytics.js]: all the options above are hacky solutions, your comment is not helping

Comment (esteeroramy) [reply public/js/analytics.js]: an `ajax` request to check the permission would be the correct thing to do here. the reason why is that this is a hack that we are relying on the default behavior of these classes, which is potential to change.

File: server.js (hunk 4 above)
Comment (ahmedqarmout2) [reply server.js]: fixed in another request

File: public/js/analytics.js (hunk 3 above)
Comment (esteeroramy) [reply public/js/analytics.js]: this is still not addressed

**Review by esteeroramy — APPROVED** [review]

**Review by Mohamed-A-Mohamed — APPROVED** [review]

**Pull request merged**